.github/workflows/main.yml

```diff
@@ -0,0 +1,59 @@
+name: main
+
+on:
+  push:
+  pull_request:
```

Comment:
Can you update this to only trigger on pushes, or pull requests, targeting the `master` branch?